.github/workflows/node-test.yml

```diff
@@ -19,6 +19,9 @@ jobs:
         with:
           node-version: ${{ matrix.node }}
           cache: 'npm'
+      - name: setup npm@^7
```

Comment:
Can be dropped, we are using Node 16 now as a default: https://github.com/nextcloud/nextcloud-event-bus/commit/6f0429d39b926c3b3b648b9f45ecb7a22f32bb64